hwy/ops/x86_128-inl.h

```diff
@@ -8575,28 +8575,72 @@ HWY_API VFromD<D> LoadExpand(MFromD<D> mask, D d,
 // ------------------------------ Additional mask logical operations
 
 #if HWY_TARGET <= HWY_AVX3
+namespace detail {
+
+template <class T, HWY_IF_LANES_LE(sizeof(T), 4)>
```

Comment:
In future let's use HWY_IF_NOT_T_SIZE or T_SIZE_ONE_OF so it's easier to search for occurrences and/or immediately obvious the constraint concerns the lane size. But no need to change now :)